Jenkinsfile

```diff
@@ -26,10 +26,9 @@ pipeline {
         CODECOV_TOKEN = credentials('CODECOV_TOKEN')
         WALLETCONNECT_PROJECT_ID_STAGING = credentials('WALLETCONNECT_PROJECT_ID_STAGING')
         WALLETCONNECT_PROJECT_ID_PROD = credentials('WALLETCONNECT_PROJECT_ID_PROD')
-	WEB3AUTH_GOOGLE_CLIENT_ID_STAGING = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_STAGING')
-	WEB3AUTH_GOOGLE_CLIENT_ID_PROD = credentials('WEB3AUTH_GOOGLE_CLIENT_ID_PROD')
-	WEB3AUTH_GOOGLE_VERIFIER_STAGING = credentials('WEB3AUTH_GOOGLE_VERIFIER_STAGING')
-	WEB3AUTH_GOOGLE_VERIFIER_PROD = credentials('WEB3AUTH_GOOGLE_VERIFIER_PROD')
+	WEB3AUTH_GOOGLE_CLIENT_ID = credentials('WEB3AUTH_GOOGLE_CLIENT_ID')
```

Comment:
I believe we still need to differentiate between Staging/Prod for verifier and Client ID

Reply:
Even The Schema will be different in this case, no?

Reply:
Yes, we need prod & staging variants here.

Schema is depending on the client id provided by google. that is correct.

Reply:
Why do we need _PROD & _STAGING in this file? Why not handle it like the INFURA_KEY?

Reply:
Jenkinsfile is updated